_includes/footer.html

```diff
@@ -21,16 +21,22 @@ <h4><strong>{{ site.owner }}</strong>
         <br>
         <ul class="list-inline">
           <li>
-            <a href="#"><i class="fa fa-dribbble fa-fw fa-3x"></i></a>
+            <a href="#"><i class="fas fa-arrow-circle-up fa-fw fa-3x"></i></a> <!-- previously used fab fa-dribbble-->
           </li>
           <li>
-            <a href="https://facebook.com/{{ site.facebook_username }}"><i class="fa fa-facebook fa-fw fa-3x"></i></a>
+            <a href="https://facebook.com/{{ site.facebook_username }}"><i class="fab fa-facebook fa-fw fa-3x"></i></a>
           </li>
           <li>
-            <a href="https://twitter.com/{{ site.twitter_username }}"><i class="fa fa-twitter fa-fw fa-3x"></i></a>
+            <a href="https://twitter.com/{{ site.twitter_username }}"><i class="fab fa-twitter fa-fw fa-3x"></i></a>
           </li>
           <li>
-            <a href="https://github.com/{{ site.github_username }}"><i class="fa fa-github fa-fw fa-3x"></i></a>
+            <a href="https://github.com/{{ site.github_username }}"><i class="fab fa-github fa-fw fa-3x"></i></a>
+          </li>
+          <li>
+            <a href="https://t.me/osmUA/{{ site.telegram_username }}"><i class="fab fa-telegram fa-fw fa-3x"></i></a>
```

Comment:
З посилання треба прибрати `osmUA` 
```suggestion
            <a href="https://t.me/{{ site.telegram_username }}"><i class="fab fa-telegram fa-fw fa-3x"></i></a>
```